analysis/shamanenhancement/src/modules/apl/AplCheck.tsx

```diff
@@ -0,0 +1,78 @@
+import SPELLS from 'common/SPELLS';
+import { suggestion } from 'parser/core/Analyzer';
+import aplCheck, { build } from 'parser/shared/metrics/apl';
+import annotateTimeline from 'parser/shared/metrics/apl/annotate';
+import {
+  and,
+  buffMissing,
+  buffPresent,
+  buffStacks,
+  debuffMissing,
+  debuffPresent,
+  hasTalent,
+} from 'parser/shared/metrics/apl/conditions';
+
+/**
+ * Based on https://www.icy-veins.com/wow/enhancement-shaman-pve-dps-guide
+ */
+export const apl = build([
+  {
+    spell: SPELLS.LAVA_LASH,
+    condition: and(hasTalent(SPELLS.HOT_HAND_TALENT), buffPresent(SPELLS.HOT_HAND_BUFF)),
```

Comment:
rules that never apply are automatically hidden, so you can omit the `hasTalent` check here and rely on the `HOT_HAND_BUFF` never being present
```suggestion
    condition: buffPresent(SPELLS.HOT_HAND_BUFF),
```

that will clean up the display a little as well